### ISSUE TITLE
Extension no longer works in Firefox 118

### ISSUE THREAD

**Issue by coinshot** [tasn/webext-signed-pages#50]
Hi @tasn,

I love the idea of this extension and wanted to adopt it for some of my projects, however I couldn't get it to work on the latest Firefox. I'm also seeing some weird behaviour on Chrome where it would tell me some pages were signed even though they shouldn't be, which I suspect is because the icon is simply not updated.

I took a quick peek, and I found some issues with the current manifest.json. Then, when attempting to build this locally, some issues popped up due to a lot of the dependencies getting really old.

I suggest to bump the manifest to be compatible with V3, and also rework / bump the extension dependencies themselves.

Next, I was unable to get the icon to work at all with `page_action`, probably because it's missing the correct `show_matches` configuration, but got the extension to work again with a simple `browser_action`.

I can create a PR which bumps all the dependencies to the latest*, and attempts to get this working on the latest browsers again. Let me know if that works for you,and if you have any thoughts.

Thanks

See also:
- https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action
- https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles#manifest_v3_migration

*One tricky part is that I think the "minimize" extension hasn't been touched in years, and could use an up-to-date alternative, but for that the minification logic should be checked in depth to not break the old behaviour.

**Comment by rugk** [comment]
Yeah [switching to manifest v3](https://github.com/TinyWebEx/common/issues/14) would be great and ensures future compatibility.  [Here is a migration checklist](https://extensionworkshop.com/documentation/develop/manifest-v3-migration-guide/#migration-checklist).
